packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -217,10 +217,10 @@ describe.each([['petstore.oas2.json'], ['petstore.oas3.json']])('server %s', fil
     // accorging to the schema
 
     const expectedValues = {
-      'x-rate-limit': file === 'petstore.oas3.json' ? 1000 : expect.any(String),
-      'x-stats': file === 'petstore.oas3.json' ? 1500 : expect.any(String),
+      'x-rate-limit': file === 'petstore.oas3.json' ? 1000 : expect.any(Number),
+      'x-stats': file === 'petstore.oas3.json' ? 1500 : expect.any(Number),
       'x-expires-after': expect.any(String),
-      'x-strange-header': file === 'petstore.oas3.json' ? 'string' : '{}',
+      'x-strange-header': file === 'petstore.oas3.json' ? 'string' : {},
```

Comment:
This but confuses me. Why was it ‘{}’ before and why is it {} now? They both seem like funny things to happen, don’t we just want an empty string for a header?

Reply:
Good question, I actually wanted to brief/ask you about this anyway.

In OpenAPI 3 documents the schema has a separate property in the Header object (`example` or `examples`) — in OAS2 the schema is "melted" with the header definition itself.

For this reason, JSON Schema Faker is kind of confused and generates an empty object for a schema that's not really complete/defined.

I can add a small workaround to standardise the behaviour. Up to you!